test/common/testinsp/storage.py

```diff
@@ -0,0 +1,14 @@
+from yaml import safe_load
+from testinsp.base import TestInspector
+from testinsp.constants import YAML
+
+
+class DiskInfo(TestInspector):
+    store_type = YAML
+    _get_data_command = "udisksctl dump"
```

Comment:
This again falls into the "no surprise here" category. Our tests only ever work on `scsi_debug` or loop devices, their setup is in `--verbose --trace` and also in the journal. This is also redundant.

Reply:
Note: There might very well be specific tests where this is unclear, but it should then be restricted to a particular test or at most `storagelib.py`. But this should be triggered by a failure where it would have helped. Every piece of information which we attach to a failure has a cost -- humans need to trawl through it and find the things that actually matter. Even then we usually only add it temporarily in a draft PR for examination (if it doesn't reproduce locally), and in most cases take it out again.